apisix/admin/plugins.lua

```diff
@@ -60,16 +70,23 @@ function _M.get(name)
         return 200, http_plugins
     end
 
-    if not name then
-        return 400, {error_msg = "not found plugin name"}
+    local plugin
+    -- By default search through http subsystems
+    subsystem = subsystem or "http"
+
+    if subsystem == "http"  then
+        plugin = plugin_get_http(name)
     end
 
-    local plugin_name = "apisix.plugins." .. name
+    if subsystem == "stream" then
+        plugin = plugin_get_stream(name)
+    end
 
-    local ok, plugin = pcall(require, plugin_name)
-    if not ok then
+    if not plugin then
+        subsystem = subsystem or "http"
```

Comment:
This logic is duplicated with https://github.com/apache/apisix/pull/9580/files#diff-00090cde519e7280adbc73e830b92089c4e62a22057c1b7577c77dc1456e72e5R75

Reply:
removed